schemas/entity/arterialSpinLabelingMRI.schema.tpl.json

```diff
@@ -0,0 +1,89 @@
+{
+  "_type": "https://openminds.ebrains.eu/MRI/ArterialSpinLabelingMRI",
+  "_extends": "/MRI/MRIImage",
```

Comment:
we do not allow extended schemas to be their own type. we have to make a decision here.
Also not sure if all those properties belong in here or rather the scanner usage